update

jasonparser opened the pull request.

t4t5 committed:
Merge branch '0.3.3'

apzentral committed:
Adding html parameter to render as HTML.

Adding New Feature for Rendering HTML in Title and Text.

grimalschi committed:
Replace arguments.callee to function name

Because an error in strict mode: "Uncaught TypeError: 'caller', 'callee', and 'arguments' properties may not be accessed on strict mode functions or the arguments objects for calls to them".

t4t5 committed:
Merge branch 'master' of git://github.com/apzentral/sweetalert into 0.3.4

t4t5 committed:
Merge branch 'alert_confirm_issue_229' of git://github.com/carloscabral/sweetalert into 0.3.4

t4t5 committed:
Merge branch 'master' of git://github.com/mwakerman/sweetalert into 0.3.4

t4t5 committed:
Add npm file, gulp file and fix global close-function

t4t5 committed:
Allow animation to be turned off + Update example page

t4t5 committed:
Fix conflicts

t4t5 committed:
Set year on example page

t4t5 committed:
Merge branch '0.4.0'

Commit:
bug fix #251

t4t5 committed:
Better browser compatibility

t4t5 committed:
Fix merge conflict

t4t5 committed:
Merge branch 'master' of git://github.com/arthurvr/sweetalert into 0.4.1

t4t5 committed:
Merge branch 'master' of git://github.com/oitozero/sweetalert into 0.4.1

t4t5 committed:
Fix merge conflict

t4t5 committed:
Merge branch 'image_size_fix' of git://github.com/peterjosling/sweetalert into 0.4.1

t4t5 committed:
Minify files

t4t5 committed:
Merge branch '0.4.1'

peterjosling committed:
Remove redundant initialization check

getModal method now creates the DOM elements on demand - don't need to
poll until it's ready.

peterjosling committed:
Compile JS

Commit:
Possibility to disable confirm button color style manipulation

danreeves committed:
add AMD, CommonJS, and globals support. not adding everything to the window. point main to script for npm.

sushantdhiman committed:
Changed icons classes , .sa-icon .sa-error .sa-info .sa-success .sa-warning .sa-custom

sushantdhiman committed:
More classes changed. sa-tip

sushantdhiman committed:
More classes changed. .sa-tip .sa-long .sa-line .sa-fix .sa-placeholder .sa-dot .sa-body .sa-line .sa-x-mark

t4t5 committed:
Fix merge conflicts

t4t5 committed:
Merge #278

t4t5 committed:
Allow confirm button to be hidden and update docs

t4t5 committed:
Fix padding when no buttons are visible

t4t5 committed:
Merge branch '0.4.2'

review-ninja commented:
<a href="http://app.review.ninja/jasonparser/sweetalert/pull/2" target="_blank"><img src="http://app.review.ninja/24944563/pull/2/badge" alt="ReviewNinja"/>

MathRobin committed:
Update README.md

Add npm install info

t4t5 committed:
Input field WIP

Things to fix:
- Bug: If you press enter while focus is on the input, the modal will
close down
- Add a swal.showInputError method which shows an error next to the
input field based on your custom logic

cmingxu committed:
[MOD] correct a misspell

t4t5 committed:
Add input error handling for user

t4t5 committed:
Merge branch 'master' of https://github.com/cmingxu/sweetalert into 0.5.0

t4t5 committed:
Merge branch 'master' of https://github.com/MathRobin/sweetalert into 0.5.0

t4t5 committed:
Fix padding for IE

t4t5 committed:
inputType, stop scroll and update documentation

t4t5 committed:
Make allowOutsideClick call callback. Fix #299

t4t5 committed:
Bug fixes and update docs

t4t5 committed:
Add themes

t4t5 committed:
Update README

t4t5 committed:
Merge branch '0.5.0'

t4t5 committed:
Update README

Commit:
Add Placeholder support

Commit:
Fix for #320, `allowOutsideClick: false` no longer works

t4t5 committed:
Themes and allowOutsideClick

t4t5 committed:
Merge branch 'master' of https://github.com/brandfolder/sweetalert into themes

t4t5 committed:
Update example

t4t5 committed:
Use dist instead and commit .jshint

t4t5 committed:
Use ES6 syntax for imports

t4t5 committed:
Break out button interactions

t4t5 committed:
Break out keyboard events + rename files

t4t5 committed:
Finalize new file structure

t4t5 committed:
Fix persisting class bug. Close #287

toddwprice committed:
added inputDefault value

ecwyne committed:
array => object

jeremylevy committed:
Fix #105

RobertoNovelo committed:
Index.html Updated With The Timeout Callback Example

RobertoNovelo committed:
Timeout Callback Implementation

t4t5 committed:
Merge branch 'patch-1' of https://github.com/ecwyne/sweetalert into stable-1.0.0

t4t5 committed:
Upgrade gulp-sass so that i works with newer Node version

t4t5 committed:
Merge changes for Require.js support

t4t5 committed:
Fix code style and bugs for timeout callbacks

t4t5 committed:
Merge callback for timer

t4t5 committed:
Allow default value in input

t4t5 committed:
Update version in package.json

t4t5 committed:
Merge branch 'stable-1.0.0'

t4t5 committed:
Fix bower.json paths

Close #360 and close #361

dasilvacontin committed:
typo in README, 'min' -> 'mind'

epool committed:
- Loading button added to use with ajax requests.

epool committed:
- Change name method from 'sweetAlert' to 'swal'.

dpatou committed:
Mention of included animations in documentation

t4t5 committed:
Add QUnit tests and Travis

t4t5 committed:
Merge branch 'master' of https://github.com/epool/sweetalert into ajax-loading

t4t5 committed:
Fix loader animation

t4t5 committed:
Merge branch 'patch-1' of https://github.com/dpatou/sweetalert into ajax-loading

t4t5 committed:
Merge branch 'patch-1' of https://github.com/dasilvacontin/sweetalert into ajax-loading

t4t5 committed:
Bump version number

t4t5 committed:
Fix Travis test

t4t5 committed:
Add bower install to Travis

t4t5 committed:
Testing with CDN instead

t4t5 committed:
Also add jQuery as CDN

t4t5 committed:
Add build status to project README

chentsulin committed:
add babel build for commonjs entry point

chentsulin committed:
add lib to ignore

t4t5 committed:
Update NPM version

t4t5 committed:
Merge branch 'commonjs'

t4t5 committed:
Fix gitignore

t4t5 committed:
Update NPM version

chentsulin committed:
export default sweetAlert

t4t5 committed:
Add export default

caseyWebb committed:
Support ES6 Callbacks

[ref] #463

kevPo committed:
Removed clear option that Internet Explorer puts in inputs.

tkuijer committed:
#461 Fix info icon dit position for internet explorer

thompsonemerson committed:
double quotes within double quotes

hermanzhu committed:
fix exception when use node. 'window is not defined'

hermanzhu committed:
run gulp scripts

hermanzhu committed:
fix grammatical mistake

hermanzhu committed:
run gulp scripts

t4t5 committed:
Merge branch 'patch-1' of https://github.com/thompsonemerson/sweetalert into v1.1.4

t4t5 committed:
Bug fixes

t4t5 committed:
Resolve merge conflicts

t4t5 committed:
Merge branch 'remove-clear-field' of https://github.com/kevPo/sweetalert into v1.1.4

t4t5 committed:
Merge branch 'master' of https://github.com/tkuijer/sweetalert into v1.1.4

t4t5 committed:
Restructure project. Use Webpack, PostCSS and TypeScript

t4t5 committed:
Add promise interface

t4t5 committed:
More types

t4t5 committed:
Handle first options

t4t5 committed:
Generate swal markup on every call

t4t5 committed:
Refactor. Add tslint

t4t5 committed:
Working button options

t4t5 committed:
Make all icons work

t4t5 committed:
Enhance buttons

t4t5 committed:
getState and update all classes

t4t5 committed:
Begin new docs and make all options available

t4t5 committed:
Add setValueFor function for awesome flexibility

t4t5 committed:
Clean up elements, input support, custom icon

t4t5 committed:
Add dangerMode

t4t5 committed:
Make modal scrollable + UI fixes

t4t5 committed:
Use Jus for docs

t4t5 committed:
Basic doc layout

t4t5 committed:
Write awesome docs

t4t5 committed:
Setup Jest tests

t4t5 committed:
Enhance presentation

t4t5 committed:
Bug fixes and simplify

t4t5 committed:
Export correctly

t4t5 committed:
Remove dist-folder

t4t5 committed:
Add dist to gitignore

t4t5 committed:
Add assets for README

t4t5 committed:
Replace gif

t4t5 committed:
Start updating README

t4t5 committed:
Realign

t4t5 committed:
Gotta love badges

t4t5 committed:
Build docs

t4t5 committed:
Merge branch 'master' of https://github.com/t4t5/sweetalert-beta into phoenix

t4t5 committed:
Create CNAME

t4t5 committed:
Add custom domain

t4t5 committed:
Merge branch 'master' of https://github.com/t4t5/sweetalert-beta into phoenix

t4t5 committed:
Remove gif

t4t5 committed:
Mobile-optimize docs page

t4t5 committed:
Mobile Safari doc bugfix

t4t5 committed:
Simplify API

t4t5 committed:
Update dependencies to reduce bundle size

t4t5 committed:
Update README.md

t4t5 committed:
Fix README

t4t5 committed:
Merge branch 'master' of https://github.com/t4t5/sweetalert-beta into phoenix

t4t5 committed:
Clean up. Add timer and closeOnEsc

t4t5 committed:
Build docs

t4t5 committed:
Update docs

t4t5 committed:
Change CNAME

t4t5 committed:
Update asset URLs

t4t5 committed:
Update Travis script

t4t5 committed:
Script fix

t4t5 committed:
Merge pull request #711 from t4t5/phoenix

2.0!

t4t5 committed:
Trigger new website

t4t5 committed:
Make website work in Safari

t4t5 committed:
Fix Safari bugs

t4t5 committed:
Fix mobile + documentation error

t4t5 committed:
Update package.json

t4t5 committed:
Fix error in docs

t4t5 committed:
Fix theming (close #714) and font-size for content (close #713)

Perlmint committed:
use more specific types.

Perlmint committed:
fix type error - should be exported

Perlmint committed:
emit declarations & sweetalert.d.ts

set types property in package.json for types resolution

Perlmint committed:
fix minor typo

Perlmint committed:
do not pack src

t4t5 committed:
Fix untranspiled const

t4t5 committed:
Merge pull request #732 from Perlmint/master

generate typescript declaration files & fix some types

t4t5 committed:
Change cdnjs to jsdelivr in docs

t4t5 committed:
Use single declaration file

t4t5 committed:
Upgrade NPM

t4t5 committed:
Make typings work

t4t5 committed:
Re-add copywebpack

Commit:
Added Polyfills, refactored some example code to make it IE11-friendly

Commit:
Added babel-standalone, reverted some of the changes

Commit:
Recompiled assets

t4t5 committed:
Merge pull request #739 from lionralfs/bugfix/ie11-issues

IE 11 Bugfixes

t4t5 committed:
Recompile

t4t5 committed:
Bump NPM

krvajal committed:
[wip] Allow multiple class names on buttons

This pull request allows the use of multiple class names when passing the `className` options to the button. The different classes should be separated by spaces.

krvajal committed:
fix typo

lionralfs committed:
Fixed styling issue with succuess icon (#712)

t4t5 committed:
Compile unminified src to dist too

t4t5 committed:
Add deprecation warnings and bump NPM

t4t5 committed:
Revert commit adding faulty unminified source

t4t5 committed:
Fix SSR for polyfills

t4t5 committed:
Bump NPM

lionralfs committed:
Merge upstream/master

aaroncolaco committed:
Add default export. Fixes #761

t4t5 committed:
Merge pull request #768 from aaroncolaco/master

Add default export. Fixes #761

lionralfs committed:
Included possibility for multiple classes via an array

lionralfs committed:
Merged existing implementation

lionralfs committed:
Simplified className implementation

lionralfs committed:
Improved readability

Commit:
fixes default for input elements

lionralfs committed:
Automatically replace newline with <br> tag

jasonday committed:
Added Accessibility attributes

Commit:
button options are optional and array allowed now

Commit:
change back earlier edits

Commit:
also case of boolean instead of buttonoptions

Commit:
have to explicitly cast some things

eek committed:
FIX: On windows, overflow scroll always shows scrollbar which is hideous, `auto` must be used to be hidden when not needed.

t4t5 committed:
Merge pull request #773 from jasonday/master

Added Accessibility attributes

t4t5 committed:
Merge pull request #744 from krvajal/patch-1

Allow multiple class names on buttons

t4t5 committed:
Merge pull request #771 from alex-osman/master

fixes default for input elements

t4t5 committed:
Merge pull request #772 from lionralfs/feature/nl2br

Automatically replace \n with <br>

t4t5 committed:
Merge pull request #777 from eek/master

FIX: Windows Scrollbars

t4t5 committed:
Merge branch 'master' into fix-typings

t4t5 committed:
Merge branch 'master' into fix-typings

t4t5 committed:
Merge pull request #774 from alex-osman/fix-typings

Fix Button Interface Typings

t4t5 committed:
Merge branch 'master' into bugfix/success-icon

t4t5 committed:
Merge branch 'master' into bugfix/success-icon

t4t5 committed:
Merge pull request #747 from lionralfs/bugfix/success-icon

Fixed styling issue with success icon (#712)

t4t5 committed:
Compile & bump NPM version

arihantdaga committed:
Namespace is optional.

Namespace is optional. Right now calling swal.close() gives error in typescript.

monkeywithacupcake committed:
Added .github/ISSUE_TEMPLATE.md (optional)

monkeywithacupcake committed:
Added call to donate after npm install (optional)

monkeywithacupcake committed:
Added backers and sponsors on the README

t4t5 committed:
Merge pull request #802 from arihantdaga/patch-1

Namespace is optional.

jsainthilaire committed:
Set namespace definition to be optional

lionralfs committed:
Merge pull request #805 from jsainthilaire/bugfix/namespace-optional-on-close

Set namespace definition to be optional

rhysd committed:
fix typos

lionralfs committed:
Merge pull request #810 from rhysd/fix-typos

Fix typos

rapito committed:
Update buttons css to use proper selector to avoid disabled .swal-buttons

rapito committed:
Use proper not selector to ignore buttons with certain attributes

Sphinxxxx committed:
Fixes #831

t4t5 committed:
Add opencollective

t4t5 committed:
OpenCollective fixes

t4t5 committed:
Remove notice from postinstall

t4t5 committed:
Merge pull request #866 from t4t5/opencollective-opencollective

Opencollective

SheetJSDev committed:
README source ASI removal

lionralfs committed:
Fix broken links in README

Also made the links absolute so they point to github.com, even when accessed from npmjs.com for instance.

t4t5 committed:
Use raw GitHub URLs in README

t4t5 committed:
Merge pull request #875 from t4t5/lionralfs-patch-1

Fix broken links in README

t4t5 committed:
Merge pull request #868 from SheetJSDev/master

README source ASI removal

t4t5 committed:
Merge pull request #861 from Sphinxxxx/patch-1

Fixes #831

t4t5 committed:
Merge pull request #855 from rapito/master

Fix sass compilation error on #854

t4t5 committed:
Bump NPM

t4t5 committed:
Update documentation for React usage

lionralfs committed:
Fix localhost link in README

t4t5 committed:
Merge pull request #877 from t4t5/lionralfs-patch-2

Fix localhost link in README

lionralfs committed:
fix pointer-events in IE8-IE10

t4t5 committed:
Merge pull request #897 from lionralfs/feature/pointer-events-ie10-fixes

Fix pointer-events in IE8-IE10

0xflotus committed:
fixed small error

CKurisu committed:
Fix Error to TypeScript Import

lionralfs committed:
Merge pull request #944 from 0xflotus/patch-1

fixed small error (typo in docs)

t4t5 committed:
Merge pull request #979 from CKurisu/master

Fix Error to TypeScript Import
